core/field_angle.js

```diff
@@ -66,6 +66,22 @@ Blockly.FieldAngle.fromJson = function(options) {
   return new Blockly.FieldAngle(options['angle']);
 };
 
+/**
+ * Editable fields usually show some sort of UI indicating they are editable.
+ * They will also be saved by the XML renderer.
+ * @type {boolean}
+ * @const
+ */
+Blockly.FieldAngle.prototype.EDITABLE = true;
```

Comment:
Since EDITABLE defaults to true on fields, you don't need to redeclare it on each of these classes.  You only need to declare it for classes that set it to false.  The inverse is true for SERIALIZABLE.

Reply:
Fixed.